### ISSUE TITLE
[Design] Fabio speech bubbles accessibility

### ISSUE THREAD

**Issue by yago1994** [AppHatchery/Fabla-Front-end#527]
# Goal
Improve the accessibility of Fabio's speech bubbles, the text size is currently not accessible or follows accessibility guidelines.

## Task
- Evaluate potential design alternatives and current issue

**Comment by Tulika-Banerjee** [comment]
Same issue created and added under To-do.
